visivo/models/model.py

```diff
@@ -1,14 +1,32 @@
-from typing import Optional
+from typing import List, Optional, Union
 from pydantic import Field, validator
 from visivo.models.base.named_model import NamedModel
 from visivo.models.base.parent_model import ParentModel
 from visivo.models.target import DefaultTarget, Target
 from .base.base_model import generate_ref_field
+from abc import ABC
 
 
-class Model(NamedModel, ParentModel):
+class Model(ABC):
     """
-    Models are queries that return base data that is used in Traces
+    Models return base data that is used in Traces
+    """
+
+
+class ScriptModel(Model, NamedModel):
+    """
+    Model that executes a script that returns csv data
```

Comment:
This is super cool. Not a blocker- but security will be an interesting consideration down the line when we give people the ability to set up scheduled updates from the cloud app.

Reply:
We would run those scripts in a docker, so the script would be inside that container.